querydsl-r2dbc/src/main/java/com/querydsl/r2dbc/JavaTypeMapping.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2015, The Querydsl Team (http://www.querydsl.com/team)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.querydsl.r2dbc;
+
+import com.google.common.primitives.Primitives;
+import com.querydsl.core.util.ReflectionUtils;
+import com.querydsl.r2dbc.types.*;
+
+import javax.annotation.Nullable;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * {@code JavaTypeMapping} provides a mapping from Class to Type instances
+ *
+ * @author tiwe
+ */
+class JavaTypeMapping {
```

Comment:
Just use the one from `querydsl-dsl` . Make it `public` if you have to.

Reply:
checked this -> so there are types for r2dbc that changed and this registry is for those types (notice the import `import com.querydsl.r2dbc.types.*;`